External/FEXCore/Source/Interface/Core/JIT/Arm64/MemoryOps.cpp

```diff
@@ -1340,6 +1340,165 @@ DEF_OP(StoreMemTSO) {
   }
 }
 
+DEF_OP(MemSet) {
```

Comment:
Just so it isn't lost or anything, maybe leave just an info or general todo comment here that mentions that MOPS instructions could be used here whenever its feasible to do so

Reply:
Added a comment at the head of the implementation.